sdk-api-src/content/ntdef/nf-ntdef-initializeobjectattributes.md

```diff
@@ -10,6 +10,7 @@ ms.keywords: InitializeObjectAttributes, InitializeObjectAttributes macro [Kerne
 ms.topic: macro
 f1_keywords:
 - ntdef/- InitializeObjectAttributes
+- "ntdef/FIELD_OFFSET"

```

Comment:
I don't think developers pressing F1 on `FIELD_OFFSET` will want to read about `InitializeObjectAttributes`. Better remove that keyword.

Reply:
I added it because I found it lying at the bottom of the markdown. Going to remove it now.

```suggestion
```